agent/rpm/pbench-agent.spec.j2

```diff
@@ -92,15 +99,29 @@ if [ $1 -eq 0 ] ;then
 fi
 
 %postun
-# if uninstalling, rather than updating, remove /%{installdir}
+# if uninstalling, rather than updating, remove /%{installdir} except id_rsa and pbench-agent.cfg
```

Comment:
We can emit a message in the `%postun` scriptlet if we'd like, but not sure that will do any good since it is out of our hands "how" the private was created.  Another key management system could be in use.

Reply:
Added a warning.